ledger_test.go

```diff
@@ -34,7 +34,7 @@ func TestLedger(t *testing.T) {
 	fmt.Printf("address: %+v\n", address)
 
 	// Get Extended Addresses
-	addresses, err := device.Addresses(10)
+	addresses, err := device.Addresses([]uint32{0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10})
```

Comment:
Probably doesn't matter... but to stay in line with the prior code... I think we should drop the `10`.
```suggestion
	addresses, err := device.Addresses([]uint32{0, 1, 2, 3, 4, 5, 6, 7, 8, 9})
```